File: examples/python/advanced/error-handling/exponential-backoff.py
retry_config = RetryStrategyConfig(
    max_attempts=5,
    initial_delay_seconds=1,    # First retry after 1 second
    max_delay_seconds=60,        # Cap at 60 seconds
    backoff_rate=2.0,            # Double delay each time: 1s, 2s, 4s, 8s, 16s...
)

File: examples/java/advanced/error-handling/exponential-backoff.java
var stepConfig = StepConfig.builder()
    .retryStrategy(RetryStrategies.exponentialBackoff(
        5,                         // max attempts
        Duration.ofSeconds(1),     // first retry after 1 second
        Duration.ofSeconds(60),    // cap at 60 seconds
        2.0,                       // double delay each time: 1s, 2s, 4s, 8s, 16s...
        JitterStrategy.FULL))
    .build();
